rslib/src/scheduler/answering/mod.rs

```diff
@@ -443,9 +443,20 @@ impl Collection {
             .storage
             .get_deck(card.deck_id)?
             .or_not_found(card.deck_id)?;
-        let config = self.home_deck_config(deck.config_id(), card.original_deck_id)?;
+        let home_deck = if card.original_deck_id.0 == 0 {
```

Comment:
Minor suggestion: Change `home_deck_config()` to `home_deck()` and use it instead to reduce some duplication.